tests/rkt_run_pod_manifest_test.go

```diff
@@ -0,0 +1,381 @@
+// Copyright 2015 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+	"testing"
+
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/ThomasRooney/gexpect"
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/appc/spec/schema"
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/appc/spec/schema/types"
+)
+
+const baseAppName = "coreos.com/rkt-inspect"
+
+func importImageAndFetchHash(t *testing.T, ctx *rktRunCtx, img string) string {
+	// Import the test image into store manually.
+	cmds := strings.Fields(ctx.cmd())
+	fetchCmd := exec.Command(cmds[0], cmds[1:]...)
+	fetchCmd.Args = append(fetchCmd.Args, "--insecure-skip-verify", "fetch", img)
+	output, err := fetchCmd.Output()
+	if err != nil {
+		t.Fatalf("Cannot read the output: %v", err)
+	}
+
+	// Read out the image hash.
+	ix := strings.Index(string(output), "sha512-")
+	if ix < 0 {
+		t.Fatalf("Unexpected result: %v, expecting a sha512 hash", string(output))
+	}
+	return strings.TrimSpace(string(output)[ix:])
+}
+
+func generatePodManifestFile(t *testing.T, manifest *schema.PodManifest) string {
+	f, err := ioutil.TempFile("", "rkt-test-manifest-")
+	if err != nil {
+		t.Fatalf("Cannot create tmp pod manifest: %v", err)
+	}
+
+	data, err := json.Marshal(manifest)
+	if err != nil {
+		t.Fatalf("Cannot marshal pod manifest: %v", err)
+	}
+	if err := ioutil.WriteFile(f.Name(), data, 0600); err != nil {
+		t.Fatalf("Cannot write pod manifest file: %v", err)
+	}
+	return f.Name()
+}
+
+func verifyHostFile(t *testing.T, tmpdir, filename string, i int, expectedResult string) {
+	filePath := path.Join(tmpdir, filename)
+	defer os.Remove(filePath)
+
+	// Verify the file is written to host.
+	if strings.Contains(expectedResult, "host:") {
+		data, err := ioutil.ReadFile(filePath)
+		if err != nil {
+			t.Fatalf("Cannot read the host file: %v", err)
+		}
+		expected := strings.Trim(strings.Trim(expectedResult, "<"), ">")
```

Comment:
Nope. It's something that the test binary (inpect) will print to the stdout. Let me just remove this.